tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/MinorDottySuite.scala

```diff
@@ -577,4 +577,118 @@ class MinorDottySuite extends BaseDottySuite {
       )
```

Comment:
will transformation like in here: https://github.com/scalameta/scalameta/issues/2029 work for tokens in `derives` ?
`trait X derives a.b.Eq` make transformation of `a.b.Eq` to `a.c.Eq`, will it be transformed correctly?

Reply:
Added some more macro magic and fixed.